Replaces the legacy DES crypt hash used in tests with a stronger
bcrypt function, as crypt() in pkgs.perl no longer supports DES

# htpasswd -nbB mufasa "Circle of Life"

diff --git a/t/lib/TestApp/htpasswd b/t/lib/TestApp/htpasswd
index 6cec784..91e5375 100644
--- a/t/lib/TestApp/htpasswd
+++ b/t/lib/TestApp/htpasswd
@@ -1 +1 @@
-mufasa:Y7hn4ncIVPOuI
+mufasa:$2y$05$.KPC4Gja9L5AxJATDQBzs.lCHkm49l/9dcoyPcJg0JhyIvsD6Gqza
